### DISCUSSION TITLE
Is it by design that CompilationEnd diagnostic analyzers no longer display tooltip/squiggles in the text editor?

### DISCUSSION THREAD

**Discussion by meadisu27** [dotnet/roslyn#68167] [Q&A] [answered]
Starting with Visual Studio v17.5 my diagnostic analyzers no longer display the tooltip/squiggly in the text editor. They are still reported in the Error List. Is this change intentional? I did read that the light bulb for Code Fixes will not display with CompilationEnd analyzers due to performance issues. Curious if the same is true now for tagging the code in the editor with squiggles?

**Comment by CyrusNajmabadi** [comment, marked as answer]
> Curious if the same is true now for tagging the code in the editor with squiggles?

Yes, the same is true.  They are far too expensive, so we cannot run them for scenarios that need to be fast (like squiggles).
